frontend_tests/node_tests/drafts.js

```diff
@@ -64,7 +76,6 @@ const timerender = zrequire("timerender");
 
 const draft_1 = {
     stream: "stream",
-    stream_id: 30,
```

Comment:
I'm not sure why this stuff with `stream_id` 30 got in this diff, note to self to look into it.